packages/catalog-realm/ai-app-generator.gts

```diff
@@ -25,6 +25,10 @@ import {
 } from '@cardstack/runtime-common';
 import { AppCard, AppCardTemplate, CardsGrid } from './app-card';
 
+const getCatalogRealm = () => {
+  return new URL('/catalog/', window.location.origin);
+};
+
```

Comment:
```suggestion
const getCatalogRealm = () => {
  return new URL('/catalog/', import.meta.url);
};

```
`import.meta.url` is the URL that this module comes from. This is something that is recognized both in the browser and in nodejs

Reply:
Excellent - thanks!